cpp/src/umap/fuzzy_simpl_set/naive.cuh

```diff
@@ -341,29 +311,26 @@ void launcher(nnz_t n,
                                        knn_dists,
                                        sigmas.data(),
                                        rhos.data(),
-                                       in.vals(),
-                                       in.rows(),
-                                       in.cols(),
+                                       out->vals(),
+                                       out->rows(),
+                                       out->cols(),
                                        n_neighbors,
                                        to_process);
   RAFT_CUDA_TRY(cudaPeekAtLastError());
+}
 
-  /*
-  if (ML::default_logger().should_log(rapids_logger::level_enum::debug)) {
-    CUML_LOG_DEBUG("Compute Membership Strength");
-    std::stringstream ss;
-    ss << in;
-    CUML_LOG_DEBUG(ss.str().c_str());
-  }
-  */
-
+template <typename value_t>
+void symmetrize(raft::sparse::COO<value_t>* in,
```

Comment:
Any reason to pass by pointer for these two COOs?

Reply:
Was just following the existing pattern of COO passed by pointer. I've updated the new methods to use references instead (but left most adjacent code to keep using pointers).